src/Corruptions.ts

```diff
@@ -232,6 +236,41 @@ const corruptionLoadoutSaveLoad = (save = true, loadout = 1) => {
     }
 }
 
+async function corruptionLoadoutGetNewName(loadout = 1) {
+    const maxChars = 9
+    const regex = /^[\x00-\xFF]*$/
+    const renamePrompt = await Prompt(
+        `What would you like to name Loadout ${loadout - 1}? ` +
+        `Names cannot be longer than ${maxChars} characters. Nothing crazy!`
+    );
+   
+    if (!renamePrompt) {
+        return Alert('Okay, maybe next time.');
+    }
+    else if (renamePrompt.length > maxChars) {
+        return Alert('The name you provided is too long! Try again.')
+    }
+    else if (!regex.test(renamePrompt)) {
+        return Alert("The Loadout Renamer didn't like a character in your name! Try something else.")
+    }
+    else {
+        player.corruptionLoadoutNames[loadout - 1] = renamePrompt
+        updateCorruptionLoadoutNames();
+    }
+}
+
+export const updateCorruptionLoadoutNames = () => {
+    const rows = getElementById<HTMLTableElement>("corruptionLoadoutTable").rows
+    for (let i = 1; i < Object.keys(player.corruptionLoadouts).length + 1; i++) {
+        const cells = rows[i + 1].cells
```

Comment:
```suggestion
    for (let i = 0; i < Object.keys(player.corruptionLoadouts).length + 1; i++) {
        const cells = rows[i + 2].cells
```
this fixes the issues